examples/README.md

```diff
@@ -3,11 +3,11 @@
 This directory contains examples of [the Pipeline CRDs](../README.md) in action.
 
 To deploy them to your cluster (after
-[installing the CRDs and running the controller](../DEVELOPMENT.md#installing-andrunning)):
+[installing the CRDs and running the controller](../DEVELOPMENT.md#getting-started)):
 
 ```bash
-kubectl apply -f examples/pipelines
 kubectl apply -f examples/
```

Comment:
Should we use `ko` here ? (like on some other `knative` projects ?)

Reply:
@vdemeester we could but I'm not sure we need to since there's nothing to build - even `ko` under the hood I think still calls out to `kubectl` (i think?) - but maybe there are some other benefits I'm missing?